dspace-server-webapp/src/test/java/org/dspace/app/rest/CollectionRestRepositoryIT.java

```diff
@@ -80,7 +80,7 @@ public void findAllTest() throws Exception {
 
 
         getClient().perform(get("/api/core/collections")
-                   .param("projection", "full"))
+                   .param("embed", CollectionMatcher.getFullEmbedsParameter()))
```

Comment:
There isn't really a "problem", but we aren't testing for a full projection. I would keep the full projection for a single test where you actually test it as the current test is passing with the "level" projection.

Reply:
if not necessary please revert it otherwise add at least a test for the full projection or copy the original test in an ignored test and report about it in a jira ticket